slicerio/tests/test_segmentation.py

```diff
@@ -133,32 +194,35 @@ def test_segmentation_create(self):
         segmentation = {
             "voxels": voxels,
             "ijkToLPS": [
-                [ 0.5, 0., 0., 10],
-                [ 0., 0.5, 0., 30],
-                [ 0., 0., 0.8, 15],
-                [ 0., 0., 0., 1. ]],
+                [0.5, 0.0, 0.0, 10],
+                [0.0, 0.5, 0.0, 30],
+                [0.0, 0.0, 0.8, 15],
+                [0.0, 0.0, 0.0, 1.0],
+            ],
             "containedRepresentationNames": ["Binary labelmap", "Closed surface"],
             "segments": [
                 {
                     "labelValue": 1,
                     "terminology": {
                         "contextName": "Segmentation category and type - 3D Slicer General Anatomy list",
                         "category": ["SCT", "123037004", "Anatomical Structure"],
-                        "type": ["SCT", "10200004", "Liver"] }
+                        "type": ["SCT", "10200004", "Liver"],
+                    },
                 },
                 {
                     "labelValue": 3,
                     "terminology": {
                         "contextName": "Segmentation category and type - 3D Slicer General Anatomy list",
                         "category": ["SCT", "123037004", "Anatomical Structure"],
                         "type": ["SCT", "39607008", "Lung"],
-                        "typeModifier": ["SCT", "24028007", "Right"] }
+                        "typeModifier": ["SCT", "24028007", "Right"],
+                    },
                 },
-            ]
+            ],
```

Comment:
Many of these extra commas are actually misleading. The commas should be removed from all places where no additional list items are allowed. Extra line breaks make the code harder to read. It would be better to let the programmer decide where to break lines, as it is a result of conscious decisions not some simplistic rules about limiting line length. Same for indentation.

I don't want to ask you to undo all line break and indentation changes, as it could be a lot of work. But let's just undo those changes in this file and make sure the formatter does not try to change this anywhere (so I can later revert annoying changes).

Reply:
I can just undo the changes and disable the ruff-format hook, since you obviously don't like the black-style formatting.